SpywareFilter/sections/allowlist.txt

```diff
@@ -1228,10 +1228,14 @@ goodwillfinds.com#%#//scriptlet("set-constant", "_etmc", "emptyArr")
 @@||sephora-track.inside-graph.com/ig.js$domain=sephora.com
 ! https://github.com/AdguardTeam/AdguardFilters/issues/140678
 ! https://github.com/AdguardTeam/AdguardFilters/issues/117423
-@@||csm.cxpublic.com/Shueisha.js$domain=wpb.shueisha.co.jp
-@@||scdn.cxense.com/cx.cce.js$domain=wpb.shueisha.co.jp
+@@||csm.cxpublic.com/Shueisha.js$domain=wpb.shueisha.co.jp|sportiva.shueisha.co.jp
+@@||scdn.cxense.com/cx.cce.js$domain=wpb.shueisha.co.jp|sportiva.shueisha.co.jp
 @@||cxense.com/cx.js$domain=shueisha.co.jp
 @@||api.cxense.com/public/widget/data?json=$domain=shueisha.co.jp
+! https://github.com/AdguardTeam/AdguardFilters/issues/177378
+@@||googletagmanager.com/gtm.js$domain=sportiva.shueisha.co.jp
+@@||csm.cxpublic.com/Shueisha/domain/sportiva.shueisha.co.jp.js$domain=sportiva.shueisha.co.jp
+@@||cdn.cookielaw.org/scripttemplates/otSDKStub.js$domain=sportiva.shueisha.co.jp
```

Comment:
Spyware filter is not blocking cookielaw (or is it showing up for you in filtering log when cookie filter is disabled?

Reply:
most likely duplicated